src/libs/colors.js

```diff
@@ -63,5 +63,6 @@ export default {
   ],
 
   slate: '#8399a4',
-  brick: '#c35b69'
+  brick: '#c35b69',
+  lighten: 'rgba(255, 255, 255, 0.2)'
```

Comment:
I don't like having this one singular alpha color filter. I'd like to see the places that use this declare their colors absolutely.